Products/CMFPlone/controlpanel/browser/usergroups_groupdetails.py

```diff
@@ -2,18 +2,35 @@
 from plone.base import PloneMessageFactory as _
 from plone.protect import CheckAuthenticator
 from Products.CMFCore.utils import getToolByName
-from Products.CMFPlone.controlpanel.browser.usergroups import (
-    UsersGroupsControlPanelView,
-)
+from Products.CMFPlone.controlpanel.browser.usergroups import UsersGroupsControlPanelView
 from Products.statusmessages.interfaces import IStatusMessage
 
-
 class GroupDetailsControlPanel(UsersGroupsControlPanelView):
+    def extract_invalid_emails(self, emails):
+        """Validate a list of email addresses."""
+        if isinstance(emails, str):
+            emails = [e.strip() for e in emails.split('\n') if e.strip()]
```

Comment:
```suggestion
            emails = [email.strip() for email in emails.split('\n') if email.strip()]
```
Also does it make sense to split by newline? I do not think so...